core/profiles/standard/standard.install

```diff
@@ -339,6 +318,181 @@ function standard_install() {
   );
   field_create_instance($instance);
 
+  $instance = array(
+    'field_name' => 'field_image',
+    'entity_type' => 'node',
+    'label' => 'Image',
+    'bundle' => 'card',
+    'description' => st('Upload an image to go with this card.'),
```

Comment:
This description should be removed. It doesn't add any value (only restates node type and field name, already on the page) and adds clutter to the node form.